### ISSUE TITLE
[A11Y] Action bar items in overflow menu of PageHeader have several accessibility violations

### ISSUE THREAD

**Issue by LaureenG** [carbon-design-system/ibm-cloud-cognitive#1789]
## What package(s) are you using?

- [x] Carbon for IBM Products (or Carbon for IBM Cloud & Cognitive)
- [ ] Legacy/CDAI
- [ ] Legacy/Security

## Detailed description

> Describe in detail the issue you're having.

When running the IBM Equal Access Accessibility Checker on the "Page header with actions toolbar" [storybook](https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-components-page-header-pageheader--with-actions-toolbar) there are multiple accessibility violations:
- Element "button" should not be focusable within the subtree of an element with an 'aria-hidden' attribute with value 'true'
- Accessible name does not match or contain the visible label text
- The <span> element has the id "[object Object]--item-label" that is already in use
- The 'id' "[object, Object]--item-label" specified for the ARIA property 'aria-describedby' value is not valid

We have the same issue in our code.

> Is this issue related to a specific component?

Yes, PageHeader's actionBarItems. Only when they are gathered in the overflow menu.

> What did you expect to happen? What happened instead? What would you like to
> see changed?

No more accessibility violations.

> What browser are you working in?

Firefox 91.6.1esr.
Chrome Version 99.0.4844.51 (Official Build) (x86_64)

> What version of the @carbon/ibm-products (or @carbon/ibm-cloud-cognitive)
> package are you using?

Both:
`"@carbon/ibm-cloud-cognitive": "^0.89.0"` (on an old release)
`"@carbon/ibm-products": "~1.6.0"` (our current work)
Still reproducible on latest version (1.10.0) of IBM Products storybook.

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

IBM Automation Decision Services (designer). We must be compliant accessibility level 3 for the end of March 2022.

## Steps to reproduce the issue

1. Go to the storybook: https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-components-page-header-pageheader--with-actions-toolbar
2. Hide the side bar to avoid noise (too many accessibility violations on it) 
3. Run the IBM Equal Access Accessibility Checker (Chrome/Firefox extension) with rule set "Latest Deployment" / accessibility guidelines "IBM Accessibility" 
4. There is no accessibility violations on action bar items
5. Reduce your browser window until two of the four action bar items end up in the overflow menu
6. Click on the overflow menu icon to open it
7. Run the IBM Equal Access Accessibility Checker again
8. There are now multiple accessibility violations

## Additional information

- Screenshots or code

![Screenshot 2022-03-15 at 20 19 18](https://user-images.githubusercontent.com/6155178/158457623-367afd36-917d-4faf-a7b4-0dc9a0428aa1.png)

![Screenshot 2022-03-15 at 20 37 07](https://user-images.githubusercontent.com/6155178/158457707-1643c05f-0c9a-4308-b50e-759bf907b1af.png)

**Comment by lee-chase** [comment]
Not able to address the right/left scroll buttons which can be shown by tabs as this needs an update to Carbon .

Raised issue https://github.com/carbon-design-system/carbon/issues/11109

**Comment by FrancoisTrible** [comment]
@lee-chase should we close this ?

**Comment by elycheea** [comment]
Closing as stale.
